### ISSUE TITLE
Documentation

### ISSUE THREAD

**Issue by Betryx** [gin-gonic/website#196]
the documentation doesnt describe any of the functions, classes or methods in it.
The only thing that the documentation gives you, is examples which you have to understand by yourself ehich isnt the point.
A documentation must be descriptive, clear, show every single aspect of the package WITH expamples which this one isnt doing.
The documentation is poorly written and unclear.
You should consider refactoring the documentation abt the framework

**Comment by Azevedo-Erick** [comment]
I agree, and if it is interesting for the project to internationalize the documentation, I would like to help translate it into Brazilian Portuguese

**Comment by lloy0076** [comment]
https://github.com/gin-gonic/gin/blob/master/docs/doc.md is better than https://gin-gonic.com/docs/ which is almost entirely devoid of anything but the examples and quick start.

It's honestly NOT a good start to using the framework.

**Comment by appleboy** [comment]
See the new website: https://github.com/gin-gonic/website/pull/289
